test/test-apimatic-core/union_type_tests/any_of_test.rb

```diff
@@ -248,10 +248,9 @@ def test_deserialize_nil_morning_type_any_of
     deserialized_morning = ApiHelper.json_deserialize(_morning_json)
     _any_of = _any_of.validate(deserialized_morning)
     actual_morning = _any_of.deserialize(nil)
-    expected_morning = nil
 
     assert _any_of.is_valid
-    assert_equal(expected_morning, actual_morning, 'Actual did not match the expected.')
+    assert_nil actual_morning
```

Comment:
why we are making changes in any-of tests area? Is it refactoring or will it cause any behavioral changes

Reply:
It is an enhancement that we can do against the testing framework that we are using currently which is minitest.